server/lib/resolvers/resolverBuilder.js

```diff
@@ -0,0 +1,60 @@
+const _ = require('lodash')
+const resolverBuilders = require('./builders')
```

Comment:
This file is already resolverBuilder.js, but in the file something else is named as resolverBuilder.

I would rename this file. Maybe to "resolverMapper" or something.